src/test/unit/algorithms/Cells4Test.cpp

```diff
@@ -560,3 +562,13 @@ TEST(Cells4Test, testEqualsOperator) {
     ASSERT_TRUE(cells1 == cells2);
   }
 }
+
+
+TEST(Cells4Test, testPerformanceInputSizeImpact) {
```

Comment:
There's a couple of nice hotgym changes. 
Should I mark this known-failing test as SKIPPED_ and we can merge this? 
The issue would still remain open.